pin/gc/gc.go

```diff
@@ -66,7 +66,7 @@ func GC(ctx context.Context, bs bstore.GCBlockstore, ls dag.LinkService, pn pin.
 					err := bs.DeleteBlock(k)
 					if err != nil {
 						errors = true
-						output <- Result{Error: &CouldNotDeleteBlockError{k, err}}
+						output <- Result{Error: &ErrCouldNotDeleteBlock{k, err}}
```

Comment:
I think what you had originally was good. It seems the go convention for error *types* is to suffix them with the word Error.

https://golang.org/pkg/net/#DNSConfigError